### DISCUSSION TITLE
Citing ImHex in scientific publications

### DISCUSSION THREAD

**Discussion by codingchipmunk** [WerWolv/ImHex#984] [General]
Hello,

I am part of a research group that used ImHex to reverse engineer a proprietary data format used by some medical software. We are now in the process of writing a paper about our project and would like to know if there is some preferred way you would like us to cite the repo and authors. Usually, citations are done using the full names of the authors. We could try to submit the paper with only the GH usernames as authors, but I am uncertain if the editors will like that. A last resort would be to just put a link to the repo in a footnote, as opposed to listing it in the references section.

**Comment by WerWolv** [comment]
Hey!
First of all thanks a lot for using ImHex in this context. I'm happy to give you my full name through an email or discord so you can use it in your Paper. You can contact me through hey@werwolv.net or on Discord using the handle WerWolv#1337
